src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -100,8 +100,7 @@ IOktaApi oktaApi
 		}
 
 		oktaApi.AddSession( sessionId );
-		string? userId = await oktaApi.GetCurrentUserIdAsync( sessionId );
```

Comment:
Should I remove the oktaApi.GetCurrentUserIdAsync command since its now unused?

Reply:
Yeah if a function has no current use and no planned future use then just remove it.